lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -33,28 +33,35 @@ import ErrorModal from './ErrorModal';
  * @prop {GroupSet[]|null} groupSets
  * @prop {(id: string|null) => void} onInput
  * @prop {string|null} selectedGroupSetId
+ * @prop {string} productFamily - Which product (canvas, blackboard...) we are running in.
```

Comment:
This is a string but it could be declared a type/enum in config.js.

The values there are currently all over the place:

https://github.com/hypothesis/lms/blob/main/lms/product/product.py#L10

we could send over the names of that enum instead of the values to be more consistent between them.

Reply:
> This is a string but it could be declared a type/enum in config.js.

I think it would be wise to indicate at least where the source of truth is for possible values.